src/stories/Library/card-grid/card-grid.scss

```diff
@@ -0,0 +1,129 @@
+// The gap we want between the items.
+// The reason we need to define it explicity is because we need it for a
+// wrapper margin hack.
+$_card-grid-gap: $s-lg;
+
+.card-grid {
```

Comment:
I think this is missing vertical spacing

Reply:
Nope, the vertical spacing comes from `.paragraph` - if card-grid stands completely alone (outside CMS), it shouldnt have bottom margin.

https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/release/2024-4-0/src/stories/Library/paragraph/paragraph.scss